README.md

```diff
@@ -312,14 +312,28 @@ you must also configure a custom domain, and set it as the `baseUrl` prop in the
 
 You can find an example react app in the [examples folder](./examples).
 
+### Widgets
+
+User Management Widget Example Usage:
```

Comment:
I think we should say more about widgets and user management widget in general, what it supports etc